src/taco-utils/utilHelper.ts

```diff
@@ -354,6 +354,16 @@ module TacoUtility {
 
             return args;
         }
+
+        /* tslint:disable:no-empty */
+        /**
+         * An explicit helper empty method, which can be used in scenarios like
+         * silent callbacks, catch all exceptions do nothing etc.
+         */
+        public static emptyMethod(args?: any): any {
```

Comment:
I'd go with a typing of `public static emptyMethod(args: any...): void` since all of these empty functions must be returning void, and we should allow for multiple function arities.